### DISCUSSION TITLE
Launch variable list and game compatability list...

### DISCUSSION THREAD

**Discussion by Tell-Sackett** [ruanformigoni/gameimage#97] [Ideas]
First off, I was wondering if someone could provide the "Environment Variable", "Executable Configuration Argument", or whatever it's called for launching straight into the game as opposed to stopping at the launcher menu first? I remember there was one, but I can't recall what it is, nor can I find documentation for it.

Perhaps this could make a good addition to the ReadMe. I'm aware that there are a number of backend variables for different tasks, including: debugging, the silent launch command that I am referring to above, a compression level argument during packing, etc. Having all of these listed out in the ReadMe would give testers and power-users quick access to key tools.

**Comment by ruanformigoni** [comment]
Hi @Tell-Sackett , I've updated the `README.md` with more info on this:

You can skip the GUI launcher and run a game directly with:
```
$ ./my-game.flatimage --select-index=0
```

This will launch the game with index `0`, to list valid indices, use:
```
$ ./my-game.flatimage --select-list
```

To disable the launcher completely, and open your game directly when double clicked in
the file manager, use:
```
$ ./my-game.flatimage fim-boot sh -c '/fim/static/gameimage-launcher "$@"' -- --select-index 0
```

**Comment by Tell-Sackett** [comment]
Cool, thanks!
